samples/crypto/test_sample/README.rst

```diff
@@ -0,0 +1,258 @@
+|                    | EdDSA       | `RFC 8032 - Edwards-Curve Digital Signature Algorithm (EdDSA)`_            | `RFC 8032 - Edwards-Curve Digital Signature Algorithm (EdDSA)`_            |
++--------------------+-------------+----------------------------------------------------------------------------+----------------------------------------------------------------------------+
+| Hash               | SHA256      | `NIST - SHA`_                                                              | `NIST - SHA`_                                                              |
++--------------------+-------------+----------------------------------------------------------------------------+----------------------------------------------------------------------------+
+|                    | SHA512      |                                                                            | `NIST - SHA`_                                                              |
++--------------------+-------------+----------------------------------------------------------------------------+----------------------------------------------------------------------------+
+| HMAC               | HMAC SHA256 | `NIST - HMAC`_                                                             | `NIST - HMAC`_                                                             |
++--------------------+-------------+----------------------------------------------------------------------------+----------------------------------------------------------------------------+
+|                    | HMAC SHA512 |                                                                            | `NIST - HMAC`_                                                             |
++--------------------+-------------+----------------------------------------------------------------------------+----------------------------------------------------------------------------+
+| HKDF               | HKDF SHA256 | `RFC 5869 - HMAC-based Extract-and-Expand Key Derivation Function (HKDF)`_ | `RFC 5869 - HMAC-based Extract-and-Expand Key Derivation Function (HKDF)`_ |
++--------------------+-------------+----------------------------------------------------------------------------+----------------------------------------------------------------------------+
+|                    | HKDF SHA512 |                                                                            | `RFC 5869 - HMAC-based Extract-and-Expand Key Derivation Function (HKDF)`_ |
++--------------------+-------------+----------------------------------------------------------------------------+----------------------------------------------------------------------------+
+
+
+Requirements
+************
+
+* One of the following development boards:
+
+  * |nRF9160DK|
+  * |nRF5340DK|
+  * |nRF52840DK|
+  * |nRF52DK|
+  * |nRF51DK|
+
+* something else??
```

Comment:
nRF5340DK added to this list may be a bit premature at this moment. Support will come at a later stage

Maybe a better approach is to list the devices it is supported on rather than focusing on the specific DKs

Reply:
@frkv  add nRF53 ?

Reply:
Yes. We were postponing it due to the coming CC312 additions, but what is there already works with nRF53 so please add it

Reply:
done